### PR TITLE
Add base images

### PR DESCRIPTION
**Pull request by piontec**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: "2.1"
+version: 2.1
```
Comment (marians) [inline .circleci/config.yml:1]: [Here](https://circleci.com/docs/configuration-reference/#version) it says that `version` should be a string. So better change this back to

```suggestion
version: "2.1"
```

File: .circleci/kratix-base-cli.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+version: 2.1
```
Comment (marians) [inline .circleci/kratix-base-cli.yml:1]:
```suggestion
version: "2.1"
```

File: .circleci/kratix-github-cli.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+version: 2.1
```
Comment (marians) [inline .circleci/kratix-github-cli.yml:1]:
```suggestion
version: "2.1"
```

File: .circleci/kratix-kubectl-cli.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+version: 2.1
```
Comment (marians) [inline .circleci/kratix-kubectl-cli.yml:1]:
```suggestion
version: "2.1"
```

File: kratix-base-cli/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,6 @@
+FROM alpine:3.20.1
+RUN apk update && apk add --no-cache yq-go=4.44.1-r2 git=2.45.2-r0 bash=5.2.26-r0 base64=0.5.2-r0 sed=4.9-r2 jq=1.7.1-r0 findutils=4.9.0-r5
```
Comment (marians) [inline kratix-base-cli/Dockerfile:2]: No `USER root` needed here?

Comment (marians) [inline kratix-base-cli/Dockerfile:2]: No `USER root` needed before this?

**Review by marians — COMMENTED** [review]
I don't really know how this works, so can't review a lot

**Review by marians — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (piontec) [reply .circleci/config.yml:1]: Schemas :D Somehow my vim got the schema for it and it says exactly the opposite!

File: kratix-base-cli/Dockerfile (hunk 5 above)
Comment (piontec) [reply kratix-base-cli/Dockerfile:2]: No, as `alpine` is `root` by default.

**Pull request merged**